build.gradle

```diff
@@ -52,12 +52,12 @@ def integTestDependOnJniLib = false
 if (!usingRemoteCluster) {
     integTestDependOnJniLib = true
     if (!usingMultiNode) {
-        apply from: 'build-tools/knnplugin-coverage.gradle'
+       // apply from: 'build-tools/knnplugin-coverage.gradle'
     }
 }
 
 ext {
-    opendistroVersion = '1.8.0'
+    opendistroVersion = '1.9.0'
```

Comment:
Because this is just upgrading ES version, ODFE version does not need to be bumped up.

Reply:
makes sense. Will revert it.